packages/types/src/index.ts

```diff
@@ -1,7 +1,7 @@
 import type { EventEmitter } from 'node:events'
-import type { State, StoreApi } from 'zustand'
+import type { StoreApi } from 'zustand'
 
-export interface Web3ReactState extends State {
```

Comment:
also why this change?

Reply:
> An interface can only extend an object type or intersection of object types with statically known members.ts(2312)

+ 

> 'State' is deprecated. Use `unknown` instead of State.

which in our case means that we can simply drop it.

Reply:
i'm guessing this was only surfaced because of fixing the type errors. i'm okay with this one going in here, but it does help to review code changes vs infra changes separately as much as possible

Reply:
I did run into this error in #750 too, if you look into the diff, it is fixed there as well. My goal for this PR was to ensure there are no extraneous type errors, hence I decided to apply those changes in a single batch. We can split it for the next time!